cmd/sys-mgmt-executor/main.go

```diff
@@ -58,109 +53,36 @@ func main() {
 	flag.Parse()
 
 	// Setup Logging
-	LoggingClient = logger.NewClient(SystemManagementExecutorKey, EnableRemote, LoggingTarget, LogLevel)
+	LoggingClient := logger.NewClient(SystemManagementExecutorKey, EnableRemote, LoggingTarget, LogLevel)
```

Comment:
Question -- `EnableRemote` is a constant set to `false`. Should this be the case? If the other services in the platform are logging remotely, shouldn't this one as well? Refer to `setLoggingTarget()` function defined elsewhere for what this entails. Currently the executor doesn't have a configuration, but maybe it should.

Reply:
Good observation. So the `sys-mgmt-executor` was conceived as being an eminently lean executable, without any "frills", much more so than the other executable that reside in packages parallel to the one housing it. But given how it (i.e. the `sys-mgmt-executor`) has grown (a bit, at least) in scope, to now include the handling of `metrics` (in addition to its primary duties of starting, stopping, and restarting services), I do need to revisit the rationale to take into account this feedback 👍

Reply:
To keep the executor as thin as possible, it is my intention to remove the logging to stdout and replace it with log entries being returned as part of the JSON result.  Thus, instead of mixing log entries with a JSON result (as is currently the case), the executor will only return a JSON result.  If the executor's log entries are important, the caller can log them through the normal mechanism on the executor's behalf.